tests/test_rocksdb_options.rs

```diff
@@ -171,6 +171,18 @@ fn set_compression_options_zstd_max_train_bytes() {
     }
 }
 
+#[test]
+#[cfg(feature = "zstd")]
```

Comment:
Shouldn't the method itself, as well as other similar methods, be hidden under the `zstd` feature?

Reply:
I guess it doesn't make sense because a compression type can be `None`, I believe. The question is, what would happen if we choose something different rather than `Zstd` or `None`? I guess it would be good to check it in a runtime:
```rust
pub fn set_wal_compression_type(&mut self, t: DBCompressionType) {
    match t {
        DBCompressionType::Zstd | DBCompressionType::None => unsafe {
            ffi::rocksdb_options_set_wal_compression(self.inner, t as c_int);
        },
        _ => unimplemented!(),
    }
}
```

Reply:
My IMHO: We should not throw unimplemented here because it is how RocksDB works. If something is not supported with compression, it will throw a message in the LOG file.

But it's better for DX, in the case when someone writes an unsupported compression type. 

Anyway, I've updated PR with `unimplemented`.

Ty